### ISSUE TITLE
H sign in the spinal cord

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11361]
**Preferred term label:**
H sign in the spinal cord

**Synonyms**

**Definition (free text, please give PubMed ID)**
A longitudinally extensive T2 hyperintense spinal cord lesion from C6 to T10, predominantly involving the gray matter and revealed in the spinal cord MRI

All of these findings can be seen in MOGAD, multiple sclerosis, and other neuroimmunological diseases.

The references supporting these labels are as follows:

 PMID: 40327177; 39318613

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001442
- label: Spinal cord H sign
- parents: [Abnormal spinal cord morphology (HP:0002143)](https://hpo.jax.org/app/browse/term/HP:0002143)
- pmid: PMID:40327177|PMID:39318613
- def: The H sign can be ascertained by spinal magnetic resonance imaging (MRI) and represents spinal cord myelitis isolated to the gray matter. A longitudinally extensive T2 hyperintense spinal cord lesion from C6 to T10, predominantly involving the gray matter. The appearance of the anomaly is said to resemble the letter H.
- comment: The H sign can be observed with Myelin oligodendrocyte glycoprotein antibody-associated disease (MOGAD), multiple sclerosis, and other neuroimmunological diseases.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11361
- synonyms: H sign in the spinal cord(NONE)
